client/bindings/python/iota_client/client.py

```diff
@@ -4,13 +4,96 @@
 from iota_client._high_level_api import HighLevelAPI
 from iota_client._utils import Utils
 from json import dumps
+import humps
+from datetime import timedelta
 
 class IotaClient(NodeCoreAPI, NodeIndexerAPI, HighLevelAPI, Utils):
-    def __init__(self, client_config=None):
+    __default = object()
+    def __init__(
+        self,
+        node = __default, 
```

Comment:
Why have this node?

Reply:
I actually thought about it. I started "mirroring" the Rust builder system to see which parameters "can" be set in Rust.

And therefore I added `node`. Another argument for it, is that most people are probably used to a syntax like:
```python
client = IotaClient('url')
#or
client = IotaClient(node='url')
```

instead of

```python
client = IotaClient(['url'])
#or
client = IotaClient(node=['url'])
```

But on the other hand maybe we should promote using the quorum feature more anyway (as this should be best practice (in crypto) anyway),

Or just make a single parameter that accepts both 🤔 

So I will think about changing it 👍

Reply:
Reimplemented this part. Now node can be: `string | array of strings | Node | array of Node`